vendors/pc/boards/windows/aws_tests/config_files/FreeRTOSConfig.h

```diff
@@ -171,7 +171,7 @@ extern void vLoggingPrint( const char * pcMessage );
  * results in the wired network being used, while setting
  * configNETWORK_INTERFACE_TO_USE to 2 results in the wireless network being
  * used. */
-#define configNETWORK_INTERFACE_TO_USE       ( 0L )
```

Comment:
This change shouldn't get checked in, as it could lead to the adapter failing silently

Reply:
Good catch. Fixed